src/Transformer/DateTimeTransformerFactory.php

```diff
@@ -79,7 +79,7 @@ private function isDateTimeType(Type $type): bool
         return true;
     }
 
-    private function getClassName(Type $type): ?string
+    private function getClassName(Type $type): string
```

Comment:
...buuuuut I don't understand this change since `Type::getClassName()` can return null :sweat_smile: 

if we kinda now that `$sourceType` represents an object, thanks to `isDateTimeType()`, maybe we can inform PHPStan?
```php
/**
 * @phpstan-assert-if-true !null $type->getClassName()
 */
private function isDateTimeType(Type $type): bool
```

Reply:
I didn't know about this trick with phpstan ! It seems really nice, thanks for sharing :pray